.github/workflows/release.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Publish to github releases
         uses: ncipollo/release-action@v1
         with:
-          artifacts: "app/build/outputs/apk/release/**"
+          artifacts: "app/build/outputs/bundle/release/**"
```

Comment:
As now we've decided to create a normal apk based multi-arc release, this change should be reverted.